src/main/java/uk/gov/cshr/civilservant/domain/Quiz.java

```diff
@@ -1,24 +1,65 @@
 package uk.gov.cshr.civilservant.domain;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import lombok.Data;
-
-import javax.persistence.*;
+import java.time.LocalDateTime;
 import java.util.HashSet;
 import java.util.Set;
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
 
-@Data
+@Builder
 @Entity
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class Quiz {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @JsonIgnore
     private Long id;
 
+    @Column
+    private String name;
+
     @OneToOne
     private Profession profession;
 
-    @OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    @OneToMany(mappedBy = "quiz",fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    @JsonManagedReference
     private Set<Question> questions = new HashSet<>();
+
+    @CreationTimestamp
+    private LocalDateTime createdOn;
+
+    @UpdateTimestamp
+    private LocalDateTime updatedOn;
+
+    @Column
+    String result;
```

Comment:
Is this the result of the quiz, the overall mark, how will it be represented?

Reply:
Possibly an enum of PASS,FAIL,ATTEMPTED. We still haven't got to the point where we calculate the result in code(Learner-journey, I believe LC-494).

Reply:
I'm thinking of kicking out this field when I do the MI reporting story. Not needed, unless we have a reporting story to filter out pass/fail criteria.